static/app/views/settings/account/notifications/notificationSettingsByType.tsx

```diff
@@ -74,6 +75,21 @@ class NotificationSettingsByType extends AsyncComponent<Props, State> {
     ];
   }
 
+  componentDidMount() {
+    trackAdvancedAnalyticsEvent('notification_settings.tuning_page_viewed', {
+      organization: null,
+      notification_type: this.props.notificationType,
+    });
+  }
+
+  trackTuningUpdated = (tuningFieldType: string) => {
```

Comment:
It doesn't look like you need a bound function, as you don't pass this method into other objects.